### DISCUSSION TITLE
阿里云盘封杀第三方网盘，会影响alist吗？

### DISCUSSION THREAD

**Discussion by chenskiro** [AlistGo/alist#3383] [General]
新闻内容 :

阿里云盘封杀第三方客户端，包括小白羊、CloudDrive等均无法使用。

[来源https://ourl.co/97273](https://ourl.co/97273)

**Comment by BlankLiLei** [comment]
我的视频播放已经寄了

**Reply by theblueearl** [reply]
我昨晚整理了一晚上的影音库，早上起来结果infuse播放失败看不了视频了。。。看来应该是这个封杀第三方访问网盘的原因，期待开发者能有较好地解决解决方案

**Reply by hui9464** [reply]
我用infuse也是播放失败，网页播放，提示加载失败。

**Reply by Dashan-IZ** [reply]
他们说不是修复了吗？为什么我还是添加不了啊
![image](https://user-images.githubusercontent.com/78125220/219056040-f31e6bd5-a1d8-4bee-a6e9-be10cb1fef7d.png)

**Comment by H3nryM4** [comment]
有没有可能阿里云盘的意思是联系第三方客户端开发者进行升级

**Reply by Dashan-IZ** [reply]
想多了，就是这样在封杀你们，都不去官方app了都用token，官方免费给你提供服务器和磁盘空间？？都用第三方了，官方怎么设置会员赚钱

**Reply by csy0413** [reply]
问题是他官方要是给tv端出个官方客户端也行啊………………官方他自己也是怕版权大棒一棒子砸下来

**Reply by simpleandstupid** [reply]
> 问题是他官方要是给tv端出个官方客户端也行啊………………官方他自己也是怕版权大棒一棒子砸下来

官方有做TV 版（只能播放视频，画质没有原画，1080p要会员）

**Comment by yanjunhui** [comment]
登新的 open API 稳定了  估计就要加密封杀以前的接口了

**Reply by ghost** [reply]
很难搞定
